src/SAWScript/Proof.hs

```diff
@@ -1256,10 +1257,11 @@ solverTheorem ::
   Pos ->
   Text ->
   NominalDiffTime ->
-  IO Theorem
+  IO TheoremDB
```

Comment:
Most of these functions return a pair of `(Theorem, TheoremDB)`, but this one only returns a `TheoremDB`. Is there a reason to break with the convention here?

Reply:
There is an excellent reason: I forgot to change this function. I'll fix that.